### PR TITLE
docs: improves batch reporter docs.

### PR DESCRIPTION
**Pull request by jcchavezs**
Ping @adriancole

### CODE REVIEW

File: packages/zipkin/src/batch-recorder.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ class BatchRecorder {
    * @constructor
    * @param {Object} options
    * @property {Logger} logger logs the data to zipkin server
-   * @property {number} timeout timeout for span in microseconds
+   * @property {number} timeout timeout for reporting spans in **microseconds**
```
Comment (adriancole) [inline packages/zipkin/src/batch-recorder.js]: timeout after which an unfinished span is flushed to zipkin in microseconds

File: packages/zipkin/src/batch-recorder.js [hunk 2]
```diff
@@ -82,7 +82,7 @@ class BatchRecorder {
           this._writeSpan(id, span);
         }
       });
-    }, 1000);
+    }, 1000); // runs every second
```
Comment (adriancole) [inline packages/zipkin/src/batch-recorder.js]: // every second, this will flush to zipkin any spans that have timed out

**Comment by jcchavezs** [comment]
@adriancole added more advices. PTAL

**Comment by adriancole** [comment]
thanks @jcchavezs

**Pull request merged**